bundles/org.openhab.binding.novafinedust/README.md

```diff
@@ -0,0 +1,78 @@
+# NovaFineDust Binding
+
+This binding is for the fine dust sensor (PM Sensor) from Nova Fitness. Currently only one model is supported, the SDS011.
+
+It basically implements the protocol specified in [this document](https://cdn.sparkfun.com/assets/parts/1/2/2/7/5/Laser_Dust_Sensor_Control_Protocol_V1.3.pdf).
+One can measure the PM 2.5 and PM 10 values with this device. It comes very handy for detecting air pollution like neighbors firing their oven with wet wood etc. so one can deactivate the ventilation system. 
+
+## Supported Things
+
+There is only one Thing for this binding is `SDS011`.
+
+## Discovery
+
+There is no automatic discovery. The Thing has to be added manually via the `.things` configuration file or via (Paper) UI.
+
+## Thing Configuration
+
+There are 2 different working modes for the `SDS011` thing: Reporting and Polling.
+
+### Reporting
+
+This is the preferred mode and thus also configured as a default. In this mode the sensor wakes up every `reportingInterval` minutes, performs a measurement for 30 seconds and sleeps for `reportingInterval` minus 30 seconds. Remember: According to the [datasheet](https://www-sd-nf.oss-cn-beijing.aliyuncs.com/%E5%AE%98%E7%BD%91%E4%B8%8B%E8%BD%BD/SDS011%20laser%20PM2.5%20sensor%20specification-V1.4.pdf) the sensor has a lifetime of 8000 hours. Using a 0 as `reportingInterval` will make the sensor report its data as fast as possible.
+
+### Polling
+
+If one needs data in different intervals, i.e. not as fast as possible and not in intervals that are a multiple of full minutes, polling can be configured. The `pollingInterval` parameter specifies the time in seconds when data will be polled from the sensor.
+
+In addition to the mode one has to provide the port to which the device is connected.
+
+A full overview about the parameters of the `SDS011` thing is given in the following table:
+
+| parameter name    | mandatory | description                                                                           |
+|-------------------|-----------|---------------------------------------------------------------------------------------|
+| port              | yes       | the port the sensor is connected to, i.e. /dev/ttyUSB0.                              |
+| reporting         | yes       | whether the reporting mode (value=true) or polling mode should be used.               |
```

Comment:
Above you say that "reporting" is the default mode. If there's a default, the parameter is not mandatory, but optional.